package.json

```diff
@@ -19,11 +19,11 @@
     "url": "https://github.com/FRCDashboard/FRCDashboard/issues"
   },
   "homepage": "http://frcdashboard.github.io",
-  "devDependencies": {
-    "electron": "^1.7.12",
-    "electron-packager": "^10.1.1"
-  },
   "dependencies": {
     "wpilib-nt-client": "^1.7.1"
+  },
+  "devDependencies": {
+    "electron": "^7.1.5",
```

Comment:
Is this really a dev dependency?

Reply:
Also this is a pretty major upgrade, did you check to make sure everything runs ok?

Reply:
Yes, electron and electron packager are supposed to be in devDependencies I think. Also, I just ran my fork for the workshop today and it worked fine.

Reply:
I was worried about whether this would break packaging, but it looks like this is a convention so sounds good!